website/source/docs/enterprise/index.html.md

```diff
@@ -8,19 +8,19 @@ description: |-
 
 # Consul Enterprise
 
-Consul Enterprise simplifies operations by automating workflows. It adds support
-for microservices deployments across complex network topologies. It also
-increases both scalability and resilience. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing)
-below.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) focuses on the concerns of operating Consul at 
+organizational scale by automating common operator workflows. It introduces capabilities for expanding 
+performance scalability, resiliency and cluster lifecycle. It also adds support for application and service 
+architectures across complex network topologies. If you have already purchased Consul Enterprise, please 
+see the [licensing section](#licensing) below.
 
 Features include:
 
 - [Automated Backups](/docs/enterprise/backups/index.html)
 - [Automated Upgrades](/docs/enterprise/upgrades/index.html)
 - [Enhanced Read Scalability](/docs/enterprise/read-scale/index.html)
 - [Redundancy Zones](/docs/enterprise/redundancy/index.html)
-- [Advanced Federation for Complex Network
-  Topologies](/docs/enterprise/federation/index.html)
+- [Advanced Federation for Complex Network Topologies](/docs/enterprise/federation/index.html)
```

Comment:
I like that you've specified the use case. We'd discussed a table before, did that not make the cut?

Reply:
I had some table ideas that were a bit more involved and I want to see about tackling alongside the .io page updates. I really want to do a table here - just going to be a bit more involved short term.